pkg/apis/machine/v1beta1/condition_consts.go

```diff
@@ -27,3 +27,18 @@ const (
 	// from making any further remediations.
 	TooManyUnhealthyReason = "TooManyUnhealthy"
 )
+
+const (
+	// MachineExistsCondition is set on the Machine to show whether a virtual mahcine has been created by the cloud provider.
+	MachineExistsCondition ConditionType = "MachineExists"
```

Comment:
given the language in these conditions around "machine" vs "instance", i would have expected this to be named `InstanceExistsCondition`. what's the reasoning behind calling this one "Machine" instead of "Instance" ?

Reply:
Instance seems like a good choice, I'll update this